### ISSUE TITLE
Enabled interface monitoring using host vars

### ISSUE THREAD

**Issue by sol1-matt** [sol1/icingaweb2-module-netbox#42]
Enable interface monitoring on hosts by importing lists of interfaces that can be turned into hosts vars.

There lists available should be
- Up interfaces as strings
- Down interfaces as string
- Optional custom field `icinga_vars` of type `json` that changes adds up/down lists that are a list of dicts with the `icinga_vars` being the dict value

**Comment by sol1-matt** [comment]
This feature is ready for testing on branch [42-interface-lists](https://github.com/sol1/icingaweb2-module-netbox/tree/42-interface-lists). 

Of note is the way interfaces are gotten which is by using giant filter lists, based on the id's of the devices or vm's, that have a arbitrary max length of around 1500 chars.
This has been done to speed up the import from large datasets but could result in problems if a webserver has a max url length lower than this.

**Comment by sol1-matt** [comment]
expanded the services to create dict of dicts as well as there is similar usage between these

**Comment by sol1-matt** [comment]
This was added in #45.

It was complicated because we are iterating 
- devices
- service
- var values
to generate useful data

I tried a number of different approaches to generate info and this pull request was the cleanest

It can't handle dict generation for multiple services with the same service name but that is an existing behavior and won't be fixed. Use different service names if it is important.
